.github/workflows/maven-ci.yml

```diff
@@ -27,12 +27,16 @@ permissions: {}
 jobs:
   build:
     runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
Will this ignore any build or test failures? That would not be ideal for CI.

Reply:
Thank you, you found the leftover of a test, will remove.

Reply:
Will Switch to fail-fast: false that we can run in matrix against multiple versions and nonjob will break when an other is braking